apps/ledger-live-mobile/src/screens/Assets/index.tsx

```diff
@@ -47,6 +50,16 @@ function Assets() {
     [distribution],
   );
 
+  const assetsToDisplay = useMemo(
+    () =>
+      assets.filter(
+        asset =>
+          asset.currency.type !== "TokenCurrency" ||
+          !blacklistedTokenIds.includes(asset.currency.id),
```

Comment:
for performance reason, please use a `Set` instead of an array, we need to avoid doing includes because it induces bad performance as shown by https://github.com/LedgerHQ/ledger-live/pull/5472